### PR TITLE
X1-AV-13

### PR DESCRIPTION
**Pull request by apthornwell**
Sensors created and initial schematics made

### CODE REVIEW

**Comment by jjderooy** [comment]
@apthornwell before I review everything, is there a reason there is a "First Version" and "Second Version" folder? If not can you delete the one that isn't the official schematic.

Also delete any other files such as the todo list so I don't have to look through as many files.

Thanks!

**Comment by apthornwell** [comment]
First version was just deleted, the to do list is for my leads, and that is what is in the most recent push to github, might want to cancel this pull request and do a pull request on the things with the most recent changes

File: Xenia-1-rocket-schematic/Schematic to do list.txt [hunk 1]
```diff
@@ -0,0 +1,39 @@
+General
```
Comment (jjderooy) [inline Xenia-1-rocket-schematic/Schematic to do list.txt:1]: These should be in github issues not in a text file.

**Review by jjderooy — COMMENTED** [review]
Change wires for gnd and 5V to symbols to reduce clutter please.

image

Should be:

image

Please remove the nested folders `Xenia-1-rocket-schematic\Second version\Xenia-1-rocket-schematic.sch` should be `Wiring Schematic`.

As mentioned before, removed the todo file and put that into github issues.

Change the name of the HDMI connector symbol from `HX711` to `Strain Gauge PCB`

Change the name of the rpi symbol from `ADA3708` to `rPi0w2` unless there was a specific reason it was called that that I am not aware of.

This schematic is missing the A4988 stepper drivers needed for airbrakes.

SDA pin of LSM9DS1 is not connected.

README should be updated with some brief information regarding what sensors are connected to the rpi. Aila should help with this.

Where does the power for the rpi come from? There is a battery and BMS in the avionics bay. Ask Aila for more information.

Camera has no connection. You can just do a single wire to represent the ribbon cable that connects it to the rpi.

@TechGal should black box share a common ground? Right now the schematic shows no ground or power from rpi to black box.

Add a note in the schematic for each component just saying what it does in plain english. For example, beside the bme280 put a note saying `Barometric Pressure Sensor`

Thanks.

**Comment by jjderooy** [comment]
@apthornwell can you @ me when this is ready to be reviewed again?

**Comment by TechGal** [comment]
Looks great Alex, just a few notes. 

- By going to Tools > Annotate Schematic, you can use the automated annotation feature in KiCAD to number the components, it makes it easier to refer to them, please do this
- Once the schematic is annotated, you can go to Inspect > Electrical Rules Checker and it will make sure everything is connected, going somewhere, etc (it can be a lot of information, and can be nonsensical when you make your own parts, so this is optional, just a helpful tool to know about)
- It's best to have all voltages pointing up and grounds pointing down (just best practice)
- If several wires all go to the same place, you can connect them to improve readability (ex: the 5V pins into the stepper driver can be connected)
- Any connections not being used should have the no connection flag (like you did for the A1115)
- You might want to add net labels to the output pins on the RPi to connect them to the labeled pins on the boards, just like you did for the sensors
- I'm not sure why there are so many tildas, I'm used to seeing them denote an active low pin, but that can't be the case for something connected to a positive voltage rail, is there a reason for this, or is it unknown?
- Please fill in the title block
- The potentiometer is coming a bit off the sheet, you can move it so it's on the sheet, there's lots of room
- What's going to be connected to the Enable pin for the A4988?

That's it, great job! Didn't find anything too major, really just readability and cosmetic changes

**Comment by TechGal** [comment]
The labels look great, it's much more readable now! Ok, a few comments:
 - The Raven4 altimeters connect to ematches on the apogee and main lines (so for each one there is an ematch which then connects to ground)
 - Please mark the pins that aren't being used with X's
 - Maybe it would be a good idea to put the two different views on separate pages?
 - Finish annotating the schematic
 - In the 'Outputs' section, the NC pin on the transmitter should have an X, it's a no connect
Thanks
